starter-code/index.html

```diff
@@ -3,40 +3,80 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" type="text/css" href="./css/main.css">
 </head>
 <body>
-Premium Discover Help Download
+  <nav class="menu">
+    <img class="logo" src="images/spotify-logo.png" />
+    <ul>
+      <li class="item">Premium</li>
+      <li class="item">Discover</li>
+      <li class="item">Help</li>
+      <li class="item">Download</li>
+    </ul>
+  </nav>
+  <div class="clearfix"></div>
+  <section class="first-section">
+    <div class="content">
+        <h1>Music for everyone.</h1>
+        <p class="text">Spotify is now free on mobile, tablet and computer.
+            Listen to the right music, wherever you are.</p>
+    </div>
+      
+  </section>
+  <section class="second-section">
```

Comment:
To identify an unique element, it's a good practice to use an `id` instead of a `class`